UMLRelationship.cpp

```diff
@@ -1,10 +1,9 @@
 #include "UMLRelationship.h"
 #include <string>
 
-UMLRelationship::UMLRelationship(std::string name, UMLClass& src, UMLClass& dest){
-  this->RelationshipName=name;
-  this->ClassSrc= src;
-  this->ClassDest=dest;
+UMLRelationship::UMLRelationship(std::string name, UMLClass src, UMLClass dest)
```

Comment:
- Once you've gotten this to compile run git restore on the UMLRelationship.cpp and .h files so we dont have a merge conflict later